Ical.Net/DataTypes/Duration.cs

```diff
@@ -118,7 +119,8 @@ public static Duration FromSeconds(int seconds) =>
     /// Parses the specified value according to RFC 5545.
     /// </summary>
     /// <exception cref="System.FormatException">Thrown if the value is not a valid duration.</exception>
-    public static Duration Parse(string value) => (Duration)new DurationSerializer().Deserialize(new StringReader(value));
+    public static Duration? Parse(string value) =>
```

Comment:
Unfortunately we are somewhat inconsistent here. According to the XML doc the method should throw if the passed string is not a valid duration. `DurationSerializer` throws in some cases and returns `null` in other. I'd generally suggest to throw and return a non-nullable Duration.